README.md

```diff
@@ -31,16 +31,44 @@ Clipboard Utilities
  - Windows - clip
  - Linux   - xsel
 
+Commands
+--------
+
+System copy adds vim commands for copying `:SystemCopy`,
+`:SystemCopyLine` as well as for pasting `:SystemPaste`.
```

Comment:
My purpose was to give a possibility to map system_copy and system_paste to user defined keys, not just hard-coded c-p and c-v, here's why I want to make those commands public.